int32 StaticID
int32 FormationState
